pom.xml

```diff
@@ -14,7 +14,7 @@
   <description>Translates CQL to ELM for MADiE</description>
 
   <properties>
-    <cqframework.version>2.4.0</cqframework.version>
+    <cqframework.version>2.5.0</cqframework.version>
```

Comment:
I think v2.6.0 is available. we can use it if 2.5.0 is not a strict requirement.

Reply:
@adongare Thanks a lot Ashok! I changed it to 2.6.0.